src/lib/mongoBackend/MongoCommonRegister.cpp

```diff
@@ -351,7 +351,7 @@ HttpStatusCode processRegisterContext
 
   reg.append("_id", oid);
   reg.append(REG_EXPIRATION, expiration);
-  reg.append(REG_SERVICE_PATH, servicePath == "" ? DEFAULT_SERVICE_PATH_UPDATES : servicePath);
+  reg.append(REG_SERVICE_PATH, servicePath == "" ? SERVICE_PATH_ROOT : servicePath);
```

Comment:
Default for registers (as for subscriptions) is SERVICE_PATH_ALL

Reply:
Rationale from https://fiware-orion.readthedocs.io/en/master/user/service_path/index.html

> Under that ligth, I think is a good idea that the query associated be as wider as possible in the default case, thus /# is the one to use.

> While entities belong to services and servicepaths, subscriptions and registrations belong only to the service. The servicepath in subscriptions and registrations doesn't denote sense of belonging, but is the expression of the query associated to the subscription or registration.

(Remember this was already discussed in the past. In fact, the addition to the documentation of that fragment was the result of the discussion).

Reply:
I'm realizing that this change is in the NGSIv1 logic, thus due to constant renaming... Thus, it is ok not chanching it in this PR. However, it is a bug, as it doesn't follow the functional criteria expressed in documentation.

It'd be great to mark this with a `// FIXME Px: this has to be fixed to work as in NGSIv2`

Reply:
Issue number (to be included in the FIXME mark): https://github.com/telefonicaid/fiware-orion/issues/3078

Reply:
Fixed in 88d49e6